Runtime/Impulse/CinemachineImpulseListener.cs

```diff
@@ -18,28 +18,44 @@ namespace Cinemachine
     [HelpURL(Documentation.BaseURL + "manual/CinemachineImpulseListener.html")]
     public class CinemachineImpulseListener : CinemachineExtension
     {
+        /// <summary>
+        /// When to apply the impulse reaction.  Default is Noise.  
+        /// Modify this if necessary to influence the ordering of extension effects
+        /// </summary>
+        [Tooltip("When to apply the impulse reaction.  Default is after the Noise stage.  "
+            + "Modify this if necessary to influence the ordering of extension effects")]
+        public CinemachineCore.Stage m_ApplyAfter = CinemachineCore.Stage.Aim; // legacy compatibility setting
```

Comment:
The default value here is different from what is set in Reset():
`m_ApplyAfter = CinemachineCore.Stage.Noise; // this is the default setting`

Reply:
Yes that is intentional - hence the comments